From: Ilya Dryomov <idryomov@gmail.com>
Date: Fri, 2 Dec 2016 14:01:55 +0100
Subject: libceph: remove now unused finish_request() wrapper
Git-commit: 45ee2c1d66185e5bd27702c60cce3c43fa3370d2
Patch-mainline: v4.10-rc1
References: bsc#1048228

Kill the wrapper and rename __finish_request() to finish_request().

Signed-off-by: Ilya Dryomov <idryomov@gmail.com>
Acked-by: Luis Henriques <lhenriques@suse.com>
---
 net/ceph/osd_client.c |   14 ++++----------
 1 file changed, 4 insertions(+), 10 deletions(-)

--- a/net/ceph/osd_client.c
+++ b/net/ceph/osd_client.c
@@ -1900,7 +1900,7 @@ static void submit_request(struct ceph_o
 	__submit_request(req, wrlocked);
 }
 
-static void __finish_request(struct ceph_osd_request *req)
+static void finish_request(struct ceph_osd_request *req)
 {
 	struct ceph_osd_client *osdc = req->r_osdc;
 	struct ceph_osd *osd = req->r_osd;
@@ -1922,12 +1922,6 @@ static void __finish_request(struct ceph
 	ceph_msg_revoke_incoming(req->r_reply);
 }
 
-static void finish_request(struct ceph_osd_request *req)
-{
-	__finish_request(req);
-	ceph_osdc_put_request(req);
-}
-
 static void __complete_request(struct ceph_osd_request *req)
 {
 	if (req->r_callback)
@@ -1945,7 +1939,7 @@ static void complete_request(struct ceph
 	dout("%s req %p tid %llu err %d\n", __func__, req, req->r_tid, err);
 
 	req->r_result = err;
-	__finish_request(req);
+	finish_request(req);
 	__complete_request(req);
 	complete_all(&req->r_done_completion);
 	ceph_osdc_put_request(req);
@@ -1972,7 +1966,7 @@ static void cancel_request(struct ceph_o
 	dout("%s req %p tid %llu\n", __func__, req, req->r_tid);
 
 	cancel_map_check(req);
-	__finish_request(req);
+	finish_request(req);
 	complete_all(&req->r_done_completion);
 	ceph_osdc_put_request(req);
 }
@@ -3092,7 +3086,7 @@ static void handle_reply(struct ceph_osd
 	}
 
 	if (done_request(req, &m)) {
-		__finish_request(req);
+		finish_request(req);
 		if (req->r_linger) {
 			WARN_ON(req->r_unsafe_callback);
 			dout("req %p tid %llu cb (locked)\n", req, req->r_tid);
